src/components/Layouts/RightPart.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <div class="right-part">
+    <LastUserWidget />
+  </div>
+</template>
```

Comment:
add new line after `</template>` and `<script>`

Reply:
fixed